.eslint_dictionary.json

```diff
@@ -133,6 +133,7 @@
   "submodule",
   "subpath",
   "syncable",
+  "synthing",
```

Comment:
nit: should we use `synthesizing` ?

Reply:
it's from cdk log that we catch and map the error :shrug:

Reply:
https://github.com/aws/aws-cdk/blob/bf34b6cea31c452da2f1eda49072331466994af2/packages/aws-cdk/lib/api/util/rwlock.ts#L91